client/components/Editor/index.jsx

```diff
@@ -126,11 +126,12 @@ class Editor extends Component {
                     scenarioId: scenarioData.id
                 });
 
-                if (!this.state.tabs.slides) {
-                    const tabs = this.state.tabs;
-                    tabs.slides = this.getTab('slides');
-                    this.setState({ tabs });
-                }
+                // Clear cached new scenario values from tabs
+                Object.assign(this.state.tabs, {
```

Comment:
this still needs to use setState, and should not mutate the current state object.

```
const tabs = Object.assign({}, this.state.tabs, {...});
this.setState({tabs})

Reply:
+1

Reply:
oh good call, I forgot about that. I'll fix it earlier in the file too.

Reply:
cc @rwaldron

Reply:
Oh actually never mind, that first `Object.assign` happens in the constructor, which I believe is a fins place to directly mutate state (let me know if I'm wrong though).